src/main.js

```diff
@@ -0,0 +1,131 @@
+import './styles.css'
+// const url= 'http://localhost:3000'
+const url= 'https://omer-phonebook.herokuapp.com'
+//elements
+const root = document.getElementById('root');
+
+// Creating Elements //
+//get All
+const getAllBtn = createElement('button', 'axiosBtn')
+getAllBtn.textContent = 'Get All'
+//phoneBook info
+const getPhoneInfoBtn = createElement('button', 'axiosBtn')
+getPhoneInfoBtn.textContent = 'Get Info'
+//userInfo
+const userIdInput = createElement('input', 'userIdInput')
+userIdInput.placeholder = 'Enter User ID'
+const getUserInfoBtn = createElement('button', 'axiosBtn')
+getUserInfoBtn.textContent = 'Get User Info'
+//createUser
+const createBtn = createElement('button', 'axiosBtn')
+createBtn.textContent = 'Create User'
+//deleteUser
+const deleteBtn = createElement('button', 'axiosBtn')
+deleteBtn.textContent = 'Delete User'
+
+//outputSect
+const outputSect = createElement('div', "ouputSect");
+
+root.append(getAllBtn, getPhoneInfoBtn, getUserInfoBtn , createBtn , deleteBtn, userIdInput, outputSect);
+
+// Event Listeners
+getAllBtn.addEventListener('click',async ()=>{
+    deleteForm();
+    const data = await axios.get(`${url}/api/persons`);
+    outputSect.textContent = 'Contact List';
+    const table = createTable(data.data);
+    outputSect.append(table);
+})
+getPhoneInfoBtn.addEventListener('click',async ()=>{
+    deleteForm();
+    const data = await axios.get(`${url}/info`);
+    outputSect.innerHTML = (data.data);
+})
+getUserInfoBtn.addEventListener('click',async ()=>{
+    deleteForm();
+    try {    
+        const id= userIdInput.value;
+        if(id){
+        const data = await axios.get(`${url}/api/persons/${id}`);
+        outputSect.textContent = 'Contact Details';
+        const table = createTable([data.data]);
+        outputSect.append(table);
+        } else{
+            outputSect.textContent = 'Must Enter User ID';
+        }
+    } catch (error) {
+        outputSect.textContent = "User Doesn't Exist!";
+    }
+})
+createBtn.addEventListener('click',async ()=>{
```

Comment:
this function, and some other in its vicinity, are quite long, i would suggest trying to split and shorten functions